### PR TITLE
adding example of how to alter db and adjusting log levels

### PR DESCRIPTION
**Pull request by craigkerstiens**

### CODE REVIEW

**Comment by begriffs** [comment]
There are also some uses of this in https://citus-doc-preview-pr-638.herokuapp.com/migration/transitioning.html#validating-query-migration that probably should be updated too?

File: reference/configuration.rst [hunk 1]
```diff
@@ -213,16 +220,15 @@ The supported values for this enum are:
 
 * **debug:** Logs statement at DEBUG severity level.
 
-* **log:** Logs statement at LOG severity level.
+* **log:** Logs statement at LOG severity level. Log severity level will contain both the error, followed by :code:`STATEMENT` which contains the SQL query that was run.
```
Comment (begriffs) [inline reference/configuration.rst]: I'm not sure I understand the added comment here. Is there special behavior for `multi_task_query_log_level=log` not shared by the other levels?

**Comment by begriffs** [comment]
Fixes #634

File: reference/configuration.rst (hunk 1 above)
Comment (craigkerstiens) [reply reference/configuration.rst]: Log includes the statement following it, warning it is suppressed by default on cloud.

**Pull request merged**